packages/thirdweb/src/extensions/lens/read/getProfileData.ts

```diff
@@ -0,0 +1,41 @@
+import type { BaseTransactionOptions } from "../../../transaction/types.js";
+import { getProfile } from "../__generated__/LensHub/read/getProfile.js";
+import type { LensProfileSchema } from "./type.js";
+
+/**
+ * Download user lens profile from Arweave
+ * This method does NOT give you the user handle - consider using `getFullProfileData`
+ *
+ * @important The contract here is the LensHub contract
+ * @param options
+ * @returns LensProfileSchema | null
+ * @extension LENS
+ *
+ * @example
+ * ```ts
+ * import { getProfileData } from "thirdweb/extensions/lens";
+ *
+ * const profileData = await getProfileData({ contract, profileId });
+ *
+ * if (profileData) {
+ *   console.log("Display name: ", profileData.lens.name);
+ * }
+ * ```
+ */
+export async function getProfileData(
```

Comment:
when would i want to use this vs getFullProfileData ?

Reply:
Get profile data does not return [joined at] and [lens handle] - to query those 2 would take extra requests. 

This method return everything else: name, bio, coverImage, picture, twitter url, website url
It can be helpful in case you need such data